pom.xml

```diff
@@ -1275,6 +1270,25 @@
                     <useStandardDocletOptions>true</useStandardDocletOptions>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>enforce-maven</id>
```

Comment:
since there is only one execution, is 'id' really necessary ?

Reply:
No, it could be left out. However I think since this plugin is so generic, it's helpful to have a descriptive name for the execution.

Reply:
Agree. Would 'enforce-maven-version' be more descriptive?

Reply:
Yes, that's better. Will change.